commit 1853fc823fbd25d691986497a5b77316dae59ecc
Author: Zack Middleton <zack@cloemail.com>
Date:   Sun Jan 28 20:06:55 2024 -0600

    Update screenshot in Linux appstream metadata
    
    Retook screenshot with current Qt, default title font, and at 1000x700
    size that Flathub wants.

diff --git a/desktop/moe.clover.mm3d.metainfo.xml b/desktop/moe.clover.mm3d.metainfo.xml
index 0e0bd9d..ae3b3a9 100644
--- a/desktop/moe.clover.mm3d.metainfo.xml
+++ b/desktop/moe.clover.mm3d.metainfo.xml
@@ -19,7 +19,7 @@
 
   <screenshots>
     <screenshot type="default">
-      <image type="source">https://raw.githubusercontent.com/clover-moe/screenshots/f706e51666e364f3b3a46b23094535ff836137b2/mm3d/1.png</image>
+      <image type="source">https://raw.githubusercontent.com/clover-moe/screenshots/61fa0b6616ad97b7ba39fb222b00abb1f84d2aa2/mm3d/1.png</image>
     </screenshot>
   </screenshots>
 
